daml/Tests/BinaryOption.daml

```diff
@@ -72,4 +76,11 @@ endToEnd = script do
   submit providers.bank $ exerciseCmd alice.custodyServiceCid $ Lifecycle with
     safekeepingDepositCid = corpActDepositCid -- TODO: how do we get this in practice?
     lifecycleRequestCid = lifecycleReqCid
-    fixings = Map.fromList [ ("TSLA", Map.insert (date 2020 Mar 18) 920.0 Map.empty) ]
+    fixings = Map.fromList [ ("TSLA", Map.insert expiration 920.0 Map.empty) ]
+
+  [(_, bobsOption)] <- queryFilter @AssetDeposit bob.customer (\c -> c.asset.id.label == "TSLA Mar21 910.0")
+  bobsOption.asset.id.version === 1
+  [(_, bobsMoney)] <- queryFilter @AssetDeposit bob.customer (\c -> c.asset.id.label == "USD")
+  bobsMoney.asset.quantity === 1.0 -- TODO why not 1000.0, as Bob has 1000 options (quantity = 1000.0)
```

Comment:
As per your comment, I would expect there to be an asset deposit for 1000$. Are these tests passing?

Reply:
(fixed)